### ISSUE TITLE
Apache server.conf parse error

### ISSUE THREAD

**Issue by arroinua** [letsencrypt/letsencrypt#2525]
Hi,
I was looking for clues here https://github.com/letsencrypt/letsencrypt/issues/1294#issuecomment-153581276, but nothing helps. Also, I've searched server.conf file for some most common issues like missing double quotes and case problems - nothing found. Apachectl configtest says "Syntax OK". Maybe I've missed something?

Here is the error message from console: `PluginError(('There has been an error in parsing the file (%s): %s', u'/etc/httpd/conf/plesk.conf.d/server.conf', u'Syntax error'),)`

Here is the file's content (ips was changed): [http://pastebin.com/BsNGehsa](http://pastebin.com/BsNGehsa)

@domcleal if you still providing your help in this matter, please welcome!

**Comment by domcleal** [comment]
The parse error's from the line breaks and continuations within the `<VirtualHost>` blocks, e.g.

<pre>
<VirtualHost \
    0.0.0.0:7080 \
    [00000:000:000:0000::2]:7080 \
    0.0.0.0:7080 \
    127.0.0.1:7080 \
    >
</pre>

If you remove the _last_ `\` and put the closing `>` on the same line and after `127.0.0.1:7080` then it'll work around the problem.  You can put it back after running the LE client I think.

I've fixed the problem in Augeas and will file a PR for LE shortly.

**Comment by arroinua** [comment]
Yes, I've already figured this out, but the problem is that the server.conf is not the only config file that the client is parsing along the process. After several config files where I fixed the backslash problem, client encountered with the `/etc/httpd/conf/modsecurity.d/rules/tortix/modsec/50_plesk_basic_asl_rules.conf` file, which contains 600000+ lines and lots of backslash problems.

Anyway, thanks for your help @domcleal and for PR, I hope this issue will be fixed soon!
